js/components/menuPanelItem.js

```diff
@@ -0,0 +1,71 @@
+const React = require('react');
+const AccessibleButton = require('./accessibleButton');
+const Icon = require('./icon');
+const classNames = require('classnames');
+const PropTypes = require('prop-types');
+const CONSTANTS = require('../constants/constants');
+
+const MenuPanelItem = ({
```

Comment:
Can we include some documentation/jsdocs on how this will be used and how to use this? We should probably consider including jsdocs for parent level items (classes mainly, or in this case the MenuPanelItem const) in newly created files to help with maintenance.

Reply:
Sounds good, will add